src/styles/blocks/day.scss

```diff
@@ -0,0 +1,26 @@
+.calendar__day {
+  width: $day-size;
```

Comment:
The variable `$day-size` is used here but is not defined in this file. Ensure that `$day-size` is defined in your SCSS files to avoid compilation errors.